0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
```

Comment:
The old template used "Author(s)" which matches singular and plural, which lead to the problem you noted because the "(s)" was always left in place.

The new template uses the plural form in places where a list is allowed.  This lets it show you how to format that list (separate entries by commas, not just spaces, omitting the word "and"), and you delete things you don't need.  So, if there's only one author, you would write `Author: [Some Name](https://github.com/swiftdev)` in your actual proposal.

I think the clarity is worth the small burden of asking proposal authors to delete a bit of extra text if they don't need it.